### PR TITLE
Added getStreamPos

### PR DESCRIPTION
**Pull request by unlimitedbacon**
This is equivalent to C's ftell function.

### CODE REVIEW

**Comment by unlimitedbacon** [comment]
Even though its in the part that says DO NOT MODIFY in all caps?

**Comment by matrefeytontias** [comment]
Oops, didn't see that. You were right then, sorry.

File: src/00/filestreams.asm [hunk 1]
```diff
@@ -1705,3 +1705,69 @@ _:  pop af
...
     ret
+
+;; getStreamPos [Filestreams]
+;;  Returns the current position in a file stream.
+;; Inputs:
+;;  D: stream ID
+;; Outputs:
+;;  Z: set on success, reset on failure
+;;  A: error code (on failure)
+;;  EBC: offset to start of file in bytes
```
Comment (SirCmpwn) [inline src/00/filestreams.asm]: Capitalize the first character of the descriptions (i.e. "Stream ID")

File: src/00/filestreams.asm [hunk 2]
```diff
@@ -1705,3 +1705,69 @@ _:  pop af
...
+
+; * Get section ID from File Stream Table
+; * Using DAT linked list, count the number of sections until the beginning of the file
+; * Add 256 bytes for each section and stream pointer from File Stream Table
+
+getStreamPos:
```
Comment (SirCmpwn) [inline src/00/filestreams.asm]: Can you drop the newline above this?

**Comment by SirCmpwn** [comment]
Code looks great here, just a few minor problems.

**Comment by unlimitedbacon** [comment]
Fixed

**Comment by SirCmpwn** [comment]
Thanks :)

**Pull request merged**